README.md

```diff
@@ -130,6 +130,8 @@ Use `rustc` 1.65.0 via `rustup`
 ### Environment setup, building for macOS
 
 - Install xcode, command line utils
+- SDK Version `macosx13.3`, Xcode version `>=14.3`
```

Comment:
Had to upgrade my xcode for this, so I pushed a commit specifying that version here